src/palace/manager/celery/tasks/generate_inventory_and_hold_reports.py

```diff
@@ -36,7 +36,15 @@ def eligible_integrations(
     def is_eligible(integration: IntegrationConfiguration) -> bool:
         if integration.protocol is None:
             return False
-        settings = registry[integration.protocol].settings_load(integration)
+
+        if integration.collection.parent:
+            parent_integration = integration.collection.parent.integration_configuration
+        else:
+            parent_integration = None
+
+        settings = registry[integration.protocol].settings_load(
+            integration, parent_integration
+        )
         return isinstance(settings, OPDSImporterSettings)
```

Comment:
I'd like to avoid adding an unused `parent` parameter to the `HasIntegrationConfiguration` class, since parent isn't really meaningful in that context.

Instead of that, I think, we can just make this a `issubclass` check, which is simpler and will be faster since it doesn't have to instantiate a settings object.

```suggestion
        settings_cls = registry[integration.protocol].settings_class()
        return issubclass(settings_cls, OPDSImporterSettings)
```